### PR TITLE
feat(deps): remove langchain dependency

### PR DESCRIPTION
**Pull request by Tomas2D**
Ref: #28

### CODE REVIEW

**Review by pilartomas — COMMENTED** [review]
I can still see lanchain and langsmith inside the lockfile, it is likely not removed.

File: src/files/extraction/helpers.ts [hunk 1]
```diff
@@ -184,9 +184,12 @@ export async function getExtractedChunks(file: Loaded<File>) {
   switch (extraction.backend) {
     case ExtractionBackend.WDU: {
       const text = await getExtractedText(file);
-      const splitter = new RecursiveCharacterTextSplitter({ chunkSize: 400, chunkOverlap: 200 });
-      const documents = await splitter.createDocuments([text], undefined);
-      return documents.map((doc) => doc.pageContent);
+      const splitter = recursiveSplitString(text, {
+        size: 400,
+        overlap: 200,
+        separators: ['\n\n', '\n', ' ', '']
```
Comment (Tomas2D) [inline src/files/extraction/helpers.ts:190]: same behaviour

**Review by pilartomas — CHANGES_REQUESTED** [review]
Code looks good but langchain still gets installed via `pnpm install`

**Comment by Tomas2D** [comment]
Yes, this seems to be a problem of `pnpm`. Fixed via `.npmrc`.

**Review by pilartomas — APPROVED** [review]

**Pull request merged**